src/packages/client/src/runtime/error-types.ts

```diff
@@ -90,6 +91,17 @@ export interface MissingArgError {
   atMostOne: boolean
 }
 
+/**
+ * If a user incorrectly provided null where she shouldn't have
+ */
+export interface InvalidNullArgError {
```

Comment:
We probably want to add it to the denylist for code generation

Reply:
Indeed. I'll raise a separate PR for it